packages/arb-token-bridge-ui/src/util/index.ts

```diff
@@ -53,3 +53,23 @@ export const getAPIBaseUrl = () => {
   // Resolves: next-js-error-only-absolute-urls-are-supported in test:ci
   return process.env.NODE_ENV === 'test' ? 'http://localhost:3000' : ''
 }
+
+export enum FeatureFlags {
+  Batch = 'batch'
+}
+
+export const isExperimentalFeatureEnabled = (flag?: FeatureFlags) => {
```

Comment:
I'd rather we make it explicit that you provide a feature flag, seems less error prone.

```suggestion
export const isExperimentalFeatureEnabled = (flag: FeatureFlag) => {
```

if we want to check if *any* experimental feature is enabled, we can have a separate helper `isExperimentalModeEnabled`